packages/profile-sync-controller/src/controllers/authentication/AuthenticationController.test.ts

```diff
@@ -538,6 +538,80 @@ describe('AuthenticationController', () => {
   });
 });
 
+describe('metadata', () => {
+  it('includes expected state in debug snapshots', () => {
+    const controller = new AuthenticationController({
+      messenger: createMockAuthenticationMessenger().messenger,
+      metametrics: createMockAuthMetaMetrics(),
+    });
+
+    expect(
+      deriveStateFromMetadata(
+        controller.state,
+        controller.metadata,
+        'anonymous',
+      ),
+    ).toMatchInlineSnapshot(`
+      Object {
+        "isSignedIn": false,
+      }
+    `);
```

Comment:
@mathieuartu is that ok if we don't have any `srpSessionData` showing up? (I assume it's not defined when we setup the controller like this.

Or would you like to make a small change here to add some data to have them in the inline snapshots?